src/components/bulk-email-tool/bulk-email-task-manager/BulkEmailContentHistory.jsx

```diff
@@ -108,6 +108,7 @@ function BulkEmailContentHistory({ intl }) {
         Header: '',
         Cell: ({ row }) => (
           <Button variant="link" className="px-1" onClick={() => onViewMessageClick(tableData[row.index])}>
+            <span className="sr-only"> {row.index}</span>
```

Comment:
Can you put this after the next line (so 'view message x' instead of 'x view message')? Actually from your screen capture it looks like maybe the intended space isn't making it in there before the X, so maybe <span className="sr-only">&nbsp{row.index}</span> ?

Reply:
looks like that fixed the markup 👍 :

![image](https://user-images.githubusercontent.com/16495365/185485033-24ed4152-b7d4-41f8-9f58-5db3ca8c73de.png)